tests/sentry/replays/test_organization_replay_index.py

```diff
@@ -1249,81 +1249,81 @@ def test_get_replays_memory_error(self):
                     == b'{"detail":"Replay search query limits exceeded. Please narrow the time-range."}'
                 )
 
-    def test_get_replays_dead_rage_click_cutoff(self):
```

Comment:
please follow the procedure from https://github.com/getsentry/rfcs/blob/main/text/0027-manual-disabling-of-flaky-tests.md

Reply:
@asottile-sentry done